src/CodeCoverage/Report/JsonOverview.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/*
+ * This file is part of the PHP_CodeCoverage package.
+ *
+ * (c) Jonathan Block <block.jon@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Uses json_encode to write a small json overview containing coverage stats.
+ *
+ * @category   PHP
+ * @package    CodeCoverage
+ * @author     Jonathan Block <block.jon@gmail.com>
+ * @copyright  Jonathan Block <block.jon@gmail.com>
```

Comment:
I just used the comments from other files as a template for this file. I don't need the copyright as such to be part of this file... we can change this if necessary in order to get this to be accepted.